src/mongoChangeStream.test.ts

```diff
@@ -193,20 +193,28 @@ describe('syncing', () => {
   })
 
   test('should complete initial scan', async () => {
-    const { coll, db } = await getConns()
+    const { coll, db, redis } = await getConns()
     const sync = await getSync()
     await initState(sync, db, coll)
 
     const processed = []
     const processRecords = async (docs: ChangeStreamInsertDocument[]) => {
       await setTimeout(50)
-      processed.push(...docs)
+      for (const doc of docs) {
+        // Simulate downstream mutation
```

Comment:
This could use a longer comment, so that we understand in the future why we did this. What deletes `doc._id`?

Reply:
Ah, I see now -- `doc._id` isn't important, it's just that the `processRecords` function can mutate the document in arbitrary ways, so this is just simulating that.